src/ucp/core/ucp_request.c

```diff
@@ -24,6 +24,31 @@
 
 const ucp_request_param_t ucp_request_null_param = { .op_attr_mask = 0 };
 
+static const char *ucp_request_flag_names[] = {
+    [ucs_ilog2(UCP_REQUEST_FLAG_COMPLETED)]             = "cpml",
+    [ucs_ilog2(UCP_REQUEST_FLAG_RELEASED)]              = "rls",
+    [ucs_ilog2(UCP_REQUEST_FLAG_PROTO_SEND)]            = "proto",
+    [ucs_ilog2(UCP_REQUEST_FLAG_SYNC_LOCAL_COMPLETED)]  = "s_loc_cmpl",
+    [ucs_ilog2(UCP_REQUEST_FLAG_SYNC_REMOTE_COMPLETED)] = "s_rm_cmpl",
```

Comment:
IMO it can be shorter:
```suggestion
    [ucs_ilog2(UCP_REQUEST_FLAG_SYNC_LOCAL_COMPLETED)]  = "loc_cmpl",
    [ucs_ilog2(UCP_REQUEST_FLAG_SYNC_REMOTE_COMPLETED)] = "rm_cmpl",
```
sinse "sync" must be set in together